controllers/category.js

```diff
@@ -25,10 +25,11 @@ export default class CategoryController {
     } = req;
     try {
       const returnValue = await Category.create({ categoryName: category.toLowerCase() });
-      const { dataValues: { categoryName } } = returnValue;
+      const { dataValues: { id, categoryName } } = returnValue;
       return res.status(201).json({
         success: true,
         message: 'Category successfully added.',
+        id,
```

Comment:
@chingsley what do we need the ID here for?

Reply:
The PO wants the details of the created category to be returned along with the success message. However, he did not specify that the ID should be returned so, I can take out the ID.